src/index.js

```diff
@@ -1,39 +1,108 @@
 import './style.css';
 
 const todoList = document.getElementById('todoList');
+const addListForm = document.querySelector('#form1');
+const textInput = document.getElementById('textInput');
 
-const taskData = [
-  {
-    description: 'wash the dishes',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: '  Complete To Do list project',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'Attend family meeting',
-    completed: true,
-    index: 3,
-  },
-];
+let todos = JSON.parse(localStorage.getItem('todos'));
+// display task
+const displayTask = () => {
+  let li = '';
+  if (todos) {
+    todos.forEach((todo) => {
+      const completed = todo.completed ? 'checked' : '';
+      li += `
+      <div class="checkContainer">
+      <div class="inputContainer">
+          <input id="checkBox" type="checkbox"  ${completed} data-check="${todo.index}">
+          <input id="label" data-desc="${todo.index}" value="${todo.description}">
+      </div>
+      <div class="btnContainer">
+          <button><i id="${todo.index}" class="fa fa-ellipsis-v btnDelete"></i></button>    
+      </div>
+    </div>`;
+    });
+  }
+  todoList.innerHTML = li;
+};
+displayTask();
 
-let list = '';
-taskData.forEach((data) => {
-  list += `
+// add task
+const addList = (e) => {
+  e.preventDefault();
+  const userInput = textInput.value.trim();
+  textInput.value = '';
+  if (!userInput) return;
+  if (!todos) {
+    todos = [];
+  }
+  const list = {
+    description: userInput,
+    completed: false,
+    index: todos.length + 1,
+  };
+  todos.push(list);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  displayTask();
+};
+addListForm.addEventListener('submit', addList);
 
+// remove task
+const removeList = (index) => {
+  const newlist = todos.filter((element) => element.index !== index);
+  todos.length = 0;
+  let i = 1;
+  newlist.forEach((element) => {
+    element.index = i;
+    i += 1;
+  });
+  todos.push(...newlist);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  displayTask();
+};
+todoList.addEventListener('click', (e) => {
+  if (e.target.classList.contains('btnDelete')) {
+    const index = parseInt(e.target.getAttribute('id'), 10);
+    removeList(index);
+  }
+});
```

Comment:
- [ ] As requested by the previous reviewer, make sure that you create a new file as a module where you'll create these two functionalities for your app(Add and Remove). then you'll import that new file into your `index.js` file.